docs/acme-integration.rst

```diff
@@ -156,28 +156,44 @@ configuration active, it will check for validity of the certificate, and
 about a month before the expiration date it will try to renew the certificate
 automatically.
 
-Certificate for subdomains excluding the apex domain
-----------------------------------------------------
+Example: Certificate for apex domain and subdomains
```

Comment:
Certificates are not tied to specific hosts, and can be moved between hosts as well.

Searching for "domain certificate" in Google gives me about 51M results, and "host certificate" has about 95M results.

I suppose that "host certificate" makes sense in context, especially when we talk about a wildcard certificate vs a certificate with a bunch of SANs. Although I'm not really sure in context of server vs client certificate.